Kitodo/src/main/java/org/kitodo/production/forms/BatchForm.java

```diff
@@ -161,6 +164,9 @@ public void filterBatches() {
                 currentBatches.add(batch);
             }
         }
+        for (Batch batch : this.currentBatches) {
```

Comment:
can't you create the label in "getLabel" if there is no label?
I think it's not intuitive to create the label beforehand

Reply:
Yeah, but it is different label. I don't know why, but this class creates different label in getLabel and different in toString.